index.html

```diff
@@ -1,32 +1,164 @@
+                 <ul class="card-language-container">
+                   <li class="card-language">HTML</li>
+                   <li class="card-language">CSS</li>
+                   <li class="card-language">Javascript</li>
+                 </ul>
+                 <button class="card-btn">See project</button>
+           </div>
+    </section>
+    <section class="about-contact-container">
+        <div class="about-section-one">
+         <h3 class="about-title">About Myself</h3>
+         <p class="about-bio">Hello I’m a software developer!
+             I can help you build a product , feature or website. 
+             Look through some of my work and experience! 
+             If you like what you see and have a project you need coded, don’t hesitate 
+             to contact me.
+         </p>
+         <h4 class="connect-bottom"><a class="connect-bottom" href="#">LET’S CONNECT</a></h4>
+           <ul class="icon bottom-icon">
+             <li class="github"><a href="#"><img src="/img/github.svg" alt="github"></a></li>   
+             <li class="linkdin"><a href="#"><img src="/img/Linkedin icon.svg" alt="linkedin"></a></li>
+             <li class="v"><a href="#"><img src="/img/v.svg" alt="v"></a></li>
+             <li class="twiter"><a href="#"><img src="/img/twiter.svg" alt="twiter"></a></li>
+             <li class="m"><a href="#"><img src="/img/m.svg" alt="m"></a></li>
+             
+             
+         </ul>
+         <button class="card-btn resume">Get my resume</button>
+        </div>
+        <div class="about-section-two">
+            <ul class="about-language-container">
+                <li><h2 class="language-name">Language</h2></li>
+                <li><img src="/img/Icon - Chevron - Down - Larger.svg" alt="" class="dropdown-img"></li>
+            </ul>
+            <ul class="programing-languages-container">
+              <li class="programing-languages">
+                <img src="/img/java.svg" alt="javascript" class="programing-image">
+                  <h3 class="javascript">Javascript</h3>
+              </li>
+      
+              <li class="programing-languages">
+                <img src="/img/html.svg" alt="html" class="programing-image">
+                  <h3 class="html">HTML</h3>
+              </li>
+      
+              <li class="programing-languages">
+                  <img src="/img/css.svg" alt="css" class="programing-image">
+                  <h3 class="css">CSS</h3>
+                  
+              </li>
+      
+            </ul>
+            <ul class="frameworks-container">
+                <li><h2 class="frameworks-title">Frameworks</h2></li>
+                <li><img src="img/rightArrow.png" alt=""></li>
+            </ul>
+            <img src="/img/Separator Bottom.svg" alt="" class="skills-frameworks-divider">
+            <ul class="skills-container">
+              <li><h2 class="skills-title">Skills</h2></li>
```

Comment:
- [ ] While correcting the previous issue, you have removed this breaking between the framework title and the skills one, I hope you work on fixing this

<img width="465" alt="1" src="https://github.com/ali-techworld/Portfolio-Setup/assets/77843050/bda43b3d-252e-4f17-9f7b-fc92145e770f">

HOW IT SHOULD BE
<img width="396" alt="2" src="https://github.com/ali-techworld/Portfolio-Setup/assets/77843050/acb0480d-f414-4b71-bf47-c588eb2d83f0">